packages/firestore/src/model/field_value.ts

```diff
@@ -167,6 +173,10 @@ export class NullValue extends FieldValue {
     return this.defaultCompareTo(other);
   }
 
+  byteSize(): number {
+    return 1;
```

Comment:
It's not clear to me why you chose `1`.  I'm not saying it's a bad value, but that it might be good for us to be deliberate about our approximations and document them to make it clearer the intention and allow some future person to improve them.  

It's tempting to try to be consistent with the old code... in which case, would this use `4` since the JSON-encoding is `null` which is 4 bytes?

Alternatively, if this is really meant to represent the in-memory representation, I would _guess_ (but could be wrong that at minimum, JavaScript stores values at 4-byte-aligned locations, so the minimum size of anything is probably 4 bytes.